36_ValidSudoku.java

```diff
@@ -0,0 +1,31 @@
+// Time Complexity : O(1)

```

Comment:
We are going through all elements.

Reply:
yes, but at max we'll got through 81 elements right which is constant

Reply:
Ah, I overlooked that. You are right!